nuxt.config.js

```diff
@@ -1,4 +1,7 @@
+import { resolve } from 'path'
```

Comment:
see code example in https://v3.nuxtjs.org/examples/advanced/module-extend-pages/

```js
import { resolve } from 'pathe'
```